### DISCUSSION TITLE
[packages/reusables/src/components/deprecated-ui] What's the plan for these?

### DISCUSSION THREAD

**Discussion by atanaskanchev** [founded-labs/react-native-reusables#94] [General]
Hi @mrzachnugent , first of all, awesome job! I have been looking for an easily customisable ui lib and reusables hits the right spot. Out of curiosity, the components in deprecated-ui looks perfectly legible, what's the plan for them? Shall we use them as it is?

**Comment by mrzachnugent** [comment]
Thanks, that's fantastic!

The deprecated-ui is no longer going to be updated. It was the first draft of components with little to no focus on the web. When they work on the web, the styling does not match the ui/shadcn styling and it does not adhere to the WAI-ARIA design pattern. 

For native, the styling was made with just the component on the screen. This lead to things looking funny when putting multiple components next to each other. It was missing consistency in the styling.

I left them in the repo in-case people still want to use them.

**Reply by mrzachnugent** [reply]
`deprecated-ui` is a little hidden but it is there for people to use. In the README, all component screenshots/video are for the deprecated-ui.

In the [Backlog](https://github.com/mrzachnugent/react-native-reusables?tab=readme-ov-file#backlog) of the README, I plan on improving the remaining component from `deprecated-ui`, making them universal, and add it to the `ui` components.

For those that only want to use it for Native, there is a Native Only option for copy/pasting and I will continue to do so. For example, in the alert-dialog primitive, in the installation instruction, there is a tab where you can select `Universal` or `Native Only`: https://rnr-docs.vercel.app/primitives/alert-dialog/#installation
